requirements/setup.cfg

```diff
@@ -6,13 +6,13 @@ author_email = engineering@kadalu.io
 url = https://github.com/kadalu/kadalu
 
 [options]
-python_requires = >=3.6
+python_requires = >=3.9
 
 [options.extras_require]
 # run "make gen-requirements" to generate all requirements files
 builder = pip; setuptools; prometheus-client; jinja2; requests; datetime; xxhash
-operator = kubernetes==11.0.0
-csi = googleapis-common-protos; pyxattr; grpcio
-server = pyxattr
+operator = kubernetes==11.0.0; fastapi; urllib3; uvicorn[standard]
```

Comment:
- so new dependencies are common across all components
- these are good candidates to be added in `builder` image itself 
- and builder contains `requests` also and there's no need for explicit mention of that in component images as well